packages/app/src/cli/models/extensions/specifications/transform/app_config_webhook.ts

```diff
@@ -25,15 +25,15 @@ export function transformToWebhookConfig(content: object) {
   const serverWebhooks = getPathValue(content, 'subscriptions') as NormalizedWebhookSubscription[]
   if (!serverWebhooks) return webhooks
 
-  const webhooksSubscriptionsConfig: WebhooksConfig['subscriptions'] = []
+  const webhooksSubscriptions: WebhooksConfig['subscriptions'] = []
 
   // eslint-disable-next-line @typescript-eslint/naming-convention
   for (const {uri, topic, sub_topic, ...optionalFields} of serverWebhooks) {
-    const currSubscription = webhooksSubscriptionsConfig.find((sub) => sub.uri === uri && sub.sub_topic === sub_topic)
+    const currSubscription = webhooksSubscriptions.find((sub) => sub.uri === uri && sub.sub_topic === sub_topic)
```

Comment:
this array is always empty no? what are we doing here?

Reply:
The array is being populated inside the loop but it's filtering duplicated elements,  that's why there's a `find` filter